src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge41.java

```diff
@@ -0,0 +1,98 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.owasp.wrongsecrets.RuntimeEnvironment;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Challenge;
+import org.owasp.wrongsecrets.challenges.ChallengeTechnology;
+import org.owasp.wrongsecrets.challenges.Difficulty;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Component;
+
+/** This is a challenge based on finding secret using password shucking */
+@Slf4j
+@Component
+@Order(41)
+public class Challenge41 extends Challenge {
+
+  private final String password;
+
+  public Challenge41(ScoreCard scoreCard, @Value("UEBzc3dvcmQxMjM=") String password) {
```

Comment:
Can you move this hash to application.properties instead?

Reply:
Done!